test/NLog.Extensions.Logging.Tests/LoggerTests.cs

```diff
@@ -115,6 +115,34 @@ public void TestScopePropertyDictionary()
             Assert.Equal("NLog.Extensions.Logging.Tests.LoggerTests.Runner|DEBUG|message with id and 1 parameters |Hello", target.Logs.LastOrDefault());
         }
 
+        [Fact]
+        public void TestInvalidFormatString()
+        {
+            try
+            {
+                GetRunner().Log(Microsoft.Extensions.Logging.LogLevel.Information, 0, null, "{0}{1}", "Test");
+                Assert.False(true, "Exception must be thrown");
+            }
+            catch (AggregateException ex)
+            {
+                Assert.IsType(typeof(FormatException), ex.InnerException);
+            }
+        }
+
+        [Fact]
+        public void TestInvalidFormatString2()
+        {
+            try
+            {
+                GetRunner<Runner>(new NLogProviderOptions() { CaptureMessageTemplates = false }).Log(Microsoft.Extensions.Logging.LogLevel.Information, 0, null, "{0}{1}", "Test");
+                Assert.False(true, "Exception must be thrown");
```

Comment:
why not `ex = Assert.Throws<..>` ?  (also other test)

Reply:
How does that work with AggregateException and InnerException ?

Reply:
```c#
var aggregateException  = Assert.Throws<AggregateException > (...)
Assert.IsType<FormatException>(aggregateException.InnerException) 
```
?

Reply:
Fixed